services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -1236,6 +1239,57 @@ async def _create_fmd_for_upload(
             fmd=fmd
         )
 
+    async def create_s3_export(
+        self,
+        user_id: UserID,
+        object_keys: list[StorageFileID],
+        *,
+        progress_cb: AsyncReportCB | None,
+    ) -> StorageFileID:
+        source_object_keys: set[StorageFileID] = set()
+
+        for object_key in object_keys:
+            async for meta_data_files in get_s3_client(self.app).list_objects_paginated(
+                self.simcore_bucket_name, object_key
+            ):
+                for entry in meta_data_files:
+                    source_object_keys.add(entry.object_key)
+
+        _logger.debug("will archive '%s' files", len(source_object_keys))
```

Comment:
NIONR:log context might be more informative?

Reply:
reworded the message but n need for a log context